hector_mapping/src/HectorMappingRos.cpp

```diff
@@ -372,6 +381,33 @@ bool HectorMappingRos::mapCallback(nav_msgs::GetMap::Request  &req,
   return true;
 }
 
+bool HectorMappingRos::resetMapCallback(std_srvs::Trigger::Request  &req,
+                                        std_srvs::Trigger::Response &res)
+{
+  ROS_INFO("HectorSM Reset map service called");
+  slamProcessor->reset();
+  return true;
+}
+
+bool HectorMappingRos::pauseMapCallback(std_srvs::SetBool::Request  &req,
+                                        std_srvs::SetBool::Response &res)
+{
+  if (req.data && !pause_scan_processing_)
+  {
+    res.message = "HectorSM Mapping paused";
+    ROS_INFO("%s", res.message.c_str());
+  }
+  else if (!req.data && pause_scan_processing_)
+  {
+    res.message = "HectorSM Mapping no longer paused";
+    ROS_INFO("%s", res.message.c_str());
+  }
+  pause_scan_processing_ = req.data;
+  res.success = true;
+  return true;
+}
```

Comment:
simple service callback that sets `pause_scan_processing_` to whatever was requested in `req`

Reply:
What is HectorSM supposed to mean?

Reply:
If you look at most `ROS_INFO` in this file, they all start with `HectorSM` as a prefix... 
Examples:
```
ROS_INFO("HectorSM sysMsgCallback, msg contents: %s", string.data.c_str());
ROS_INFO("HectorSM reset");
ROS_INFO("HectorSM Map service called");
```

I personally like to have a prefix in `ROS_INFO` that indicates which node is making those prints, as that allows us to know who is printing (I find it useful in large projects).

I will update this section to print `ROS_INFO` with the prefix `HectorSM`, but I won't return `HectorSM` as part of `res.message`